ivy_tests/test_ivy/test_functional/test_nn/test_losses.py

```diff
@@ -66,11 +65,11 @@ def test_cross_entropy(
         instance_method,
         fw,
         "cross_entropy",
-        true=np.asarray(true[:length], dtype=true_dtype),
-        pred=np.asarray(pred[:length], dtype=pred_dtype),
+        true=np.asarray(true, dtype=true_dtype),
```

Comment:
Just want to confirm, are you varying the shape of `pred` and `true` during data generation? This [line](https://github.com/unifyai/ivy/blob/29ed0b33ef412b1325d186bd9ef741d95c287892/ivy_tests/test_ivy/helpers.py#L1219) indicates to me that you are, but just wanted to be sure.

Reply:
I'm generating arrays of same length but st.shared has been implemented in the helpers functions for that after checking in the module, so additional checking is removed!